docs/docs/v7-upgrade-guidance.mdx

```diff
@@ -26,6 +26,22 @@ In version 6 of the CLI for Microsoft 365, for setting the list commandset, you
 
 Since the option `id` was mandatory in V6 and if your scripts are dependent on the default mandatory error output which the command would return, you may have to update your scripts accordingly.
 
+## Enhanced `flow run list` and `flow environment list` output
+
+In version 7 of the CLI for Microsoft 365, we have made an enhancement for the data returned by the `flow run list` and `flow environment list` commands. Currently, when no items were found, we would only log an output when `--verbose` was specified. This made this command very unpredictable to use in scripts. This has now been changed so an empty array will be logged when no items were found. Also, when specifying `--output json`, we would have duplicate properties in the result. These properties have been removed.
```

Comment:
Instead of `currently` we should use previously because we are  talking about the previous version.